### PR TITLE
Contract added 

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Contract is added to `/contracts`
UI is added to `/dapp`
CI Workflow is updated with format, lint, tests 

TODO:

- [x] Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,35 @@
...
 
-SigmaDAO is a web application that lets anyone create a DAO, enable them to Add Proposal, Deposit and Withdraw Tokens and also allow them to Vote.
-
-Wallets supported by the SigmaDAO are:
-
-- [AlgoSigner](https://www.purestake.com/technology/algosigner/)
-- [MyAlgo Wallet](https://wallet.myalgo.com/new-account)
-- [Wallet Connect](https://walletconnect.com/)
-
-## SigmaDAO glimpes:
```
Comment (rmeena840) [inline README.md:13]: Why are removing these?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,35 @@
...
-Follow the examples/dao [README.MD](https://github.com/scale-it/algo-builder/blob/develop/examples/dao/README.md#setup-sigma-dao) in algo-builder to setup the Sigma Dao role and table in PostgreSQL.
+To deploy and execute the above discussed functions check the `/contracts` folder and its `README.md`
 
-## Development
+### Dapp
 
-Start the development server on http://localhost:3000
-
-```bash
-yarn dev
```
Comment (rmeena840) [inline README.md:50]: It's good to have it here.

File: contracts/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+		"@algo-builder/web": "^5.0.1",
+		"algosdk": "^1.19.0"
+	},
+	"devDependencies": {
+		"eslint": "^8.10.0"
+	},
+	"scripts": {
+		"algob": "algob",
+		"lint:check": "eslint --ext .js,.ts scripts",
+		"lint": "eslint --cache --fix --ext .js,.ts scripts",
```
Comment (rmeena840) [inline contracts/package.json:17]:
```suggestion
		"lint:fix": "eslint --cache --fix --ext .js,.ts scripts",
```

File: dapp/src/utility/algod.utility.ts [hunk 4]
```diff
@@ -6,7 +6,7 @@ import { toRaw } from "vue";
 import { HttpNetworkConfig } from "@algo-builder/web/build/types";
 import { BETA_NET_URL, MAIN_NET_URL, TEST_NET_URL } from "@/constants";
 import WalletStore from "@/store/WalletStore";
-const base32 = require("hi-base32");
+import base32 from 'hi-base32'
```
Comment (rmeena840) [inline dapp/src/utility/algod.utility.ts]:
```suggestion
import base32 from "hi-base32";
```

**Review by rmeena840 — COMMENTED** [review]
Left comments. Please make sure to not break the existing functionality.

**Comment by Megha-Dev-19** [comment]
> Left comments. Please make sure to not break the existing functionality.

Yeah I tested the deploy and also the UI

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**